### PR TITLE
degrees.py done + no requirements.txt

### PR DESCRIPTION
**Pull request by m0rg1nal**
pip freeze
ERROR: Exception:
Traceback (most recent call last):
  File "C:\Users\Ваня\PycharmProjects\pythonProject4\venv\Lib\site-packages\pip\_internal\cli\base_command.py", line 160, in exc_logging_wrapper
    status = run_func(*args)
             ^^^^^^^^^^^^^^^
  File "C:\Users\Ваня\PycharmProjects\pythonProject4\venv\Lib\site-packages\pip\_internal\commands\freeze.py", line 87, in run
    for line in freeze(
  File "C:\Users\Ваня\PycharmProjects\pythonProject4\venv\Lib\site-packages\pip\_internal\operations\freeze.py", line 42, in freeze
    for dist in dists:
  ...

### CODE REVIEW

File: Functions/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+зшз
```
Comment (MarinaZelenska) [inline Functions/requirements.txt]: This file should be in the main directory for your project

File: Functions/degrees.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    else:
+        return "Please enter F or C, these are the only available units."
+
+
+try:
+    temp = input('Enter the temperature outside: ')
+    value = int(temp.split(' ')[0])
+    units = temp.split(' ')[1].lower()
+    print(convert_degrees(value, units))
+except:
```
Comment (MarinaZelenska) [inline Functions/degrees.py:15]: You should write what type exception you want to catch

File: Functions/degrees.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    print('Please enter temperature like in this example: "12 C"')
```
Comment (MarinaZelenska) [inline Functions/degrees.py:16]: You should use block if __name__ == '__main__':

**Review by MarinaZelenska — APPROVED** [review]

File: Functions/degrees.py (hunk 2 above)
Comment (m0rg1nal) [reply Functions/degrees.py:15]: Any exception. Either user entering temperature in the wrong format or not entering enough infoormation

File: Functions/degrees.py (hunk 3 above)
Comment (m0rg1nal) [reply Functions/degrees.py:16]: Isn't it used with packages only or apps with multiple files?

**Pull request merged**